### PR TITLE
renaming :matches to :is

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://github.com/w3c/csswg-drafts/issues/3258

### CODE REVIEW

File: css/selectors.json [hunk 1]
```diff
@@ -342,14 +342,14 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:link"
   },
-  ":matches": {
-    "syntax": ":matches( <selector># )",
+  ":is": {
+    "syntax": ":is( <selector># )",
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply css/selectors.json]: Sure; updated.

File: css/selectors.json [hunk 2]
```diff
@@ -342,14 +342,14 @@
...
   },
-  ":matches": {
-    "syntax": ":matches( <selector># )",
+  ":is": {
+    "syntax": ":is( <complex-selector># )",
     "groups": [
       "Pseudo-classes",
       "Selectors"
     ],
     "status": "standard",
```
Comment (ExE-Boss) [inline css/selectors.json]: Also, this is probably `experimental` now.

Comment (chrisdavidmills) [reply css/selectors.json]: Ack, forgot about that, thanks! I did set it as experimental in BCD.

File: css/selectors.json [hunk 3]
```diff
@@ -342,14 +342,14 @@
...
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:link"
   },
-  ":matches": {
-    "syntax": ":matches( <selector># )",
+  ":is": {
+    "syntax": ":is( <complex-selector># )",
```
Comment (ExE-Boss) [inline css/selectors.json]: I’m sort to bug you a third time, but I realised that this should actually be `<complex-selector-list>`, like in https://github.com/mdn/data/pull/287: https://github.com/mdn/data/blob/83365d0f7c8665c2fb143d157bf0dc47ac598cec/css/selectors.json#L354

**Comment by chrisdavidmills** [comment]
Ah, so that's the official syntax that represents a complex selector list, so we don't need to write "one or more complex selectors". Works for me. Updated.

**Comment by chrisdavidmills** [comment]
Done and merged. Thanks!

**Pull request merged**